core/src/main/java/discord4j/core/object/emoji/EmojiCustom.java

```diff
@@ -18,40 +18,38 @@
  * <p>
  * <a href="https://discord.com/developers/docs/resources/emoji#emoji-resource">Emoji Resource</a>
  */
-public abstract class Emoji implements Entity {
+public class EmojiCustom extends Emoji {
 
     /** The path for {@code Emoji} image URLs. */
     static final String EMOJI_IMAGE_PATH = "emojis/%s";
 
-    /** The gateway associated to this object. */
-    final GatewayDiscordClient gateway;
-
     /** The raw data as represented by Discord. */
     final EmojiData data;
 
-    /**
-     * Constructs a {@code Emoji} with an associated {@link GatewayDiscordClient} and Discord data.
-     *
-     * @param gateway The {@link GatewayDiscordClient} associated to this object, must be non-null.
-     * @param data The raw data as represented by Discord, must be non-null.
-     */
-    protected Emoji(GatewayDiscordClient gateway, EmojiData data) {
-        this.gateway = Objects.requireNonNull(gateway);
+    public EmojiCustom(EmojiData data) {
```

Comment:
Why change the access?

Reply:
If remember the idea was make a simple way to get this class and move the gateway to the GuildEmoji or ApplicationEmoji

Reply:
This also allow parse the Data and use the basic methods outside of the Gateway requirement moved to Guild/Application